src/config/chartConfig.tsx

```diff
@@ -88,3 +94,64 @@ export const BAR_CHART_OPTIONS: Chart.ChartOptions = {
     },
   },
 };
+
+export const OPM_BAR_CHART_OPTIONS: Chart.ChartOptions = {
```

Comment:
Move this to `appconstants`; That is where we have kept other chart options.
And see if you can use spread operator to avoid duplicate lines of code.

Reply:
seems like there are config-related changes in Adithya's PR. I will move all the config to config.js once all those changes are in place to avoid conflicts. Used spread operator in possible places